packages/mdc-textfield/README.md

```diff
@@ -202,12 +201,23 @@ apply these mixins with CSS selectors such as `.foo-text-field.mdc-text-field--i
 
 > _NOTE_: the `mdc-line-ripple-color` mixin should be applied from the not focused class `foo-text-field:not(.mdc-text-field--focused)`).
 
-#### Mixins for Text Field Box
+#### Mixins for all Text Field
```

Comment:
This heading doesn't seem quite accurate... at the very least, the label color mixin is only applicable to filled/outlined, not full-width / textarea (which don't use floating label).

Reply:
The label color mixin applies to fullwidth and textarea.